zad1/main.rb

```diff
@@ -1,13 +1,13 @@
 def sort_letters(string)
-
+  string.reverse
```

Comment:
Metoda reverse tylko 'odwraca' string, nie sortuje go alfabetycznie ;)

Reply:
Nie uważnie czytałam zadanie :(

To teraz mam pytanie: muszę wykonać zadania jeszcze raz czy już nic z tym nie robić?

Reply:
Możesz dodać nowego commita w swoim fork z poprawkami :D Ogółem to nic się nie pali, wydaje mi się ze prace domowe można spokojnie oddawać po terminie, tak czy siak to napewno zrobimy code review